src/ThemeAssistant.php

```diff
@@ -677,9 +677,14 @@ protected function parsePhpCode($_phpCode, $data)
 
 		extract($data);
 		ob_start();
-		eval('?>' . $_phpCode . '<?php ');
 
-		return ob_get_clean() ?: '';
+		try {
+			eval('?>' . $_phpCode . '<?php ');
+			return ob_get_contents() ?: '';
```

Comment:
what's the advantage of this change? output of the errors happening there?

Reply:
This ensures that the output buffer gets cleared always (even if an error occurs). Otherwise the CSS-Code will get printend in the backend HTML output like so: 
![Bildschirmfoto 2021-04-13 um 15 45 58](https://user-images.githubusercontent.com/367169/114563213-844eed00-9c6f-11eb-85cd-63a8409a8784.png)

Reply:
LGTM!

Reply:
Although it's a little bit tricky to debug error for example in the `main.css.base` 😬

Reply:
Yes, using `eval()` is wrong here, we should instead use the sass parser or something similar.